build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    alias(libs.plugins.detekt.plugin)
+    alias(libs.plugins.ktlint.plugin)
+}
+
+buildscript {
+    apply(from = "maven.gradle")
```

Comment:
It would be cool to convert these others to Kotlin DSL too, but then you'd probably have some issues reusing them unless you went for custom plugins via buildSrc which gets difficult real fast 😅 certainly not the job for this PR, but something to consider long-term anyway.

Reply:
You can try, but last time I checked, it was not possible to do the same thing. You had to use [precompiled script plugins](https://docs.gradle.org/current/userguide/custom_plugins.html) instead, which are kinda complicated and have some issues (or had?).

However, I encourage you to look into it when you find the time - maybe you discover something I missed.